api-server/src/server/boot/certificate.js

```diff
@@ -59,12 +59,12 @@ export default function bootCertificate(app) {
 
 export function getFallbackFullStackDate(completedChallenges, completedDate) {
   var chalIds = [
-    certTypeIdMap[certTypes.respWebDesign],
-    certTypeIdMap[certTypes.jsAlgoDataStruct],
-    certTypeIdMap[certTypes.frontEndDevLibsId],
-    certTypeIdMap[certTypes.dataVis2018],
-    certTypeIdMap[certTypes.apisMicroservicesId],
-    certTypeIdMap[certTypes.legacyInfosecQaId]
+    certIds.respWebDesignId,
+    certIds.jsAlgoDataStructId,
+    certIds.frontEndDevLibsId,
+    certIds.dataVis2018Id,
+    certIds.apisMicroservicesId,
+    certIds.legacyInfosecQaId
```

Comment:
We already destructure these further up. Shall we remove an extraneous lookup with:

```suggestion
    respWebDesignId,
    jsAlgoDataStructId,
    frontEndDevLibsId,
    dataVis2018Id,
    apisMicroservicesId,
    legacyInfosecQaId
```

Reply:
Even better. Thanks!